README.md

```diff
@@ -57,6 +57,12 @@ await player.publish();
 await player.stop();
 ```
 
+#### 4. Stop the camera
```

Comment:
Maybe also explain that it calls `player.stop()` then stop the local mediaStream ?

Reply:
Added on 3e549f2